src/Particle/CMakeLists.txt

```diff
@@ -53,6 +51,34 @@ target_include_directories(qmcparticle PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}")
 target_link_libraries(qmcparticle PRIVATE platform_cpu_LA)
 target_link_libraries(qmcparticle PUBLIC qmcnumerics qmcutil platform_runtime)
 
+# Workaround for broken OMP Target debug in clang as of 5d, main is more broken so
+# perhaps this problem is fixed.
```

Comment:
"As of 5d"? (supposed to be "15"? but that is main, right?)
"main is more broken, so perhaps this problem is fixed" ?

I'm not sure what this comment is trying to say.

Reply:
I'm referring to the last known not broken for compiling offload target (although broken for some tooling) clang 15 commit from https://github.com/ye-luo/llvm_recipes/blob/master/olcf_summit/llvm_20220317/upstream-script.sh